FlutterFire.md

```diff
@@ -0,0 +1,34 @@
+# FlutterFire
```

Comment:
Let's add a note about what Flutter is. Imagine this page is linked from firebase.google.com.